packages/vaadin-notification/test/renderer.test.js

```diff
@@ -1,150 +1,110 @@
-
-    it('should clear the root when renderer changed', () => {
-      for (let i = 0; i < 2; i++) {
-        notification.renderer = (root) => {
-          const generatedContent = document.createTextNode('rendered-' + i);
-          root.appendChild(generatedContent);
-        };
-        notification.opened = true;
-        expect(notification._card.textContent.trim()).to.equal('rendered-' + i);
-      }
-    });
-
-    it('should open notification when renderer is defined after notification opened', () => {
-      notification.opened = true;
-      notification.renderer = (root) => {
-        root.appendChild(rendererContent);
-      };
-      const clientRect = notification._card.getBoundingClientRect();
-      expect(clientRect.x).to.not.equal(0);
-      expect(clientRect.y).to.not.equal(0);
-      expect(clientRect.width).to.not.equal(0);
-      expect(clientRect.height).to.not.equal(0);
-      expect(clientRect.top).to.not.equal(0);
-    });
+      expect(notification._card.textContent.trim()).to.equal('rendered-' + i);
+    }
   });
 
-  describe('with template', () => {
-    let notification;
-
-    beforeEach(() => {
-      notification = fixtureSync(`
-        <vaadin-notification>
-          <template>
-            notification-content
-          </template>
-        </vaadin-notification>
-      `);
-
-      // Force sync card attaching and removal instead of waiting for the animation
-      sinon
-        .stub(notification, '_animatedAppendNotificationCard')
-        .callsFake(() => notification._appendNotificationCard());
-      sinon
-        .stub(notification, '_animatedRemoveNotificationCard')
-        .callsFake(() => notification._removeNotificationCard());
-
-      notification.open();
-    });
-
-    afterEach(() => {
-      // Close to stop all pending timers.
-      notification.close();
-      // delete singleton reference, so as it's created in next test
-      delete notification.constructor._container;
-    });
-
-    it('should fallback to render content with Templatizer when renderer is not defined', () => {
-      expect(notification._card.textContent.trim()).to.equal('notification-content');
-    });
-
-    it('should throw an error when setting a renderer if there is already a template', () => {
-      notification._observer.flush();
-      expect(() => (notification.renderer = () => {})).to.throw(Error);
-    });
+  it('should open notification when renderer is defined after notification opened', () => {
+    notification.opened = true;
+    notification.renderer = (root) => {
+      root.appendChild(rendererContent);
+    };
+    const clientRect = notification._card.getBoundingClientRect();
+    expect(clientRect.x).to.not.equal(0);
+    expect(clientRect.y).to.not.equal(0);
+    expect(clientRect.width).to.not.equal(0);
+    expect(clientRect.height).to.not.equal(0);
+    expect(clientRect.top).to.not.equal(0);
+  });
+
+  it('should clear the notification content when removing the renderer', () => {
```

Comment:
Clearing the notification content didn't work before, and I decided to make this small fix along with removing Polymer Template API.

Reply:
@web-padawan Should I repeat this fix for `20.0` as well?

Reply:
Yes, sure. Please submit a PR for `20.0` branch.

Reply:
OK, here is the PR that addresses this issue for `20.0` branch: https://github.com/vaadin/web-components/pull/361